caffe2/operators/gather_ranges_to_dense_op.h

```diff
@@ -173,42 +178,56 @@ class GatherRangesToDenseOp final : public Operator<Context> {
     CAFFE_ENFORCE_EQ(rangesDataOffset, ranges.numel());
 
     // Check whether the empty and mismatch ratio exceeded the threshold.
-    totalRanges_ += batchSize;
+    for (const auto j : c10::irange(OutputSize())) {
+      totalRanges_[j] += batchSize;
+    }
```

Comment:
Is `OutputSize()` should always be the same as totalRanges_.size why not use a range loop?
```suggestion
   assert(OputputSize() == totalRanges_.size());
    for (auto& range : totalRanges_)  {
      range += batchSize;
    }
```

Reply:
should be, based on line 46 for (const auto i : c10::irange(OutputSize()))